### PR TITLE
877 automated e2e tests

### LINKED ISSUE #877: Automate Execution of Fink-Broker E2E Tests with Science Algorithm on Linux Server and Report to GitHub Action
We need to automate the execution of end-to-end (E2E) tests for the Fink-Broker with the science algorithm on a simple Linux server. The results of these tests should be reported back to GitHub Actions for CI/CD purposes.

**Expected behaviour:**
The E2E tests should run automatically on a specified Linux server whenever there are updates or changes to the code. The test results should be successfully reported back to the GitHub Actions workflow, indicating the pass/fail status of each test.

**Actual behaviour:**
Currently, the E2E tests are not automated and require manual execution. There is no integration with GitHub Actions to report the test outcomes automatically.

**Steps to reproduce the behaviour:**

Manually execute the E2E tests on a Linux server.
Observe the results and manually record them.
There is no current automation to send these results to GitHub Actions.

**Additional context:**
Automating this process will streamline the testing workflow and improve the efficiency of the CI/CD pipeline by providing real-time feedback on the integration of the science algorithm with the Fink-Broker.

### PR DESCRIPTION
**Pull request by fjammes**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): Closes #877 

## What changes were proposed in this pull request?

Tools to run self-hosted CI on a linux server and GHA integration.

## How was this patch tested?

On fink server @lpc

### CODE REVIEW

File: bin/raw2science.py [hunk 1]
```diff
@@ -148,33 +82,43 @@ def main():
         )
 
     # Add library versions
+    if args.noscience:
+        fsvsn = "no-science"
+    else:
+        # Do not import fink_science if --noscience is set
```
Comment (JulienPeloton) [inline bin/raw2science.py:88]: I would put this comment above in the logger:

```python
if args.noscience:
  logger.debug("Do not import fink_science as --noscience is set")
  fsvsn = "no-science"
else:
  ...
```

File: doc/troubleshoot.md [hunk 2]
```diff
@@ -12,5 +12,6 @@ kubectl run -it --rm s5cmd --image=peakcom/s5cmd --env AWS_ACCESS_KEY_ID=minio -
 Interactive access:
 ```shell
 kubectl run -it --rm s5cmd --image=peakcom/s5cmd --env AWS_ACCESS_KEY_ID=minio --env  AWS_SECRET_ACCESS_KEY=minio123 --env S3_ENDPOINT_URL=https://minio.minio:443 --command -- sh
-/s5cmd --log debug --no-verify-ssl ls -H  "s3://fink-broker-online/raw/year=2020/month=01/day=01/"
-```
\ No newline at end of file
+/s5cmd --log debug --no-verify-ssl ls -H  "s3://fink-broker-online/raw/2020/01/01/"
```
Comment (JulienPeloton) [inline doc/troubleshoot.md:15]: It should be:

```
s3://fink-broker-online/raw/20200101
```

**Review by JulienPeloton — APPROVED** [review]
Thanks @fjammes , it looks good to me! Concerning the fact that there are no topics in the output, try to increase the number of alerts launched by the simulator:

https://github.com/astrolabsoftware/fink-alert-simulator/blob/dcc8ff1d1412cc67f78f5594d04b0234506d8c18/chart/templates/configmap.yaml#L40

**Comment by fjammes** [comment]
@JulienPeloton , FYI I have added a "noscience" pipeline, because the Github one is down.

**Comment by fjammes** [comment]
LGTM

**Comment by JulienPeloton** [comment]
Why did you close this PR?

**Comment by fjammes** [comment]
My mistake, I wanted to merge it. I'll do it manually

**Comment by fjammes** [comment]
My mistake, I wanted to merge it. I'll do it manually

**Comment by fjammes** [comment]
My mistake, I wanted to merge it, I have merged it manually...

**Comment by JulienPeloton** [comment]
:D

**Pull request closed without merging**